modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/CurrencySnapshotAcceptanceManager.scala

```diff
@@ -96,6 +101,8 @@ object CurrencySnapshotAcceptanceManager {
         rewards
       )
 
+      acceptedSharedArtifacts = sharedArtifactsForAcceptance
```

Comment:
IMO it should be as it was before -> using acceptance fn. it's a place where you would have to add validation later on. my comment was related to the formatting:

```
fn(
 param
)
```

vs

```
fn(param)
```

Reply:
👍🏼